### PR TITLE
[15.0][MIG] base_technical_user

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Standard migration

CC @ForgeFlow

### CODE REVIEW

**Comment by StefanRijnhart** [comment]
@HviorForgeFlow please commit the setup directory
```
Run newfiles="$(git ls-files --others --exclude-from=.gitignore)"
Please check-in the following files:
setup/base_technical_user/odoo/addons/base_technical_user
setup/base_technical_user/setup.py
Error: Process completed with exit code 1.
```

File: base_technical_user/tests/test_sudo_tech.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import SUPERUSER_ID
+from odoo.exceptions import UserError
+from odoo.tests import SavepointCase
+
+
+class SudoTechCase(SavepointCase):
```
Comment (StefanRijnhart) [inline base_technical_user/tests/test_sudo_tech.py]: Please replace by `TransactionCase`

**Review by sebalix — APPROVED** [review]

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**